packages/sdk/client-services/src/packlets/invitations/invitations-handler.test.ts

```diff
@@ -251,6 +251,7 @@ describe('InvitationHandler', () => {
     const peer = testBuilder.createPeer();
     await peer.createIdentity();
     await openAndClose(peer.echoHost, peer.dataSpaceManager);
+    await peer.echoHost.addReplicator(peer.meshEchoReplicator);
```

Comment:
weird. this change is already in main
https://github.com/dxos/dxos/commit/65feafb476454aec66aea18b44aba16b46657a6e